azurelinuxagent/ga/monitor.py

```diff
@@ -419,13 +423,15 @@ def send_cgroup_telemetry(self):
                         if value > 0:
                             report_metric(metric_group, metric_name, cgroup_name, value)
             except Exception as e:
-                logger.warn("Failed to collect performance metrics: {0} [{1}]", e, traceback.format_exc())
+                logger.warn("Monitor: failed to collect cgroups performance metrics: {0}", ustr(e))
```

Comment:
This happens fairly frequently.  Should this be a warning, or just ignored?

Reply:
I did a quick check and did not see this in our telemetry - am I missing something? Did you mean that this exception might be thrown frequently?

Reply:
For the moment I will leave this as a warning, since I am about to enable cgroups for other distros. If we see this becomes an expected condition I will move the logs to verbose.

Reply:
ack